.github/README.md

```diff
@@ -7,7 +7,7 @@
 [![Release](https://img.shields.io/github/release/tabbycatdebate/tabbycat.svg)](https://github.com/tabbycatdebate/tabbycat/releases)
 [![Crowdin](https://badges.crowdin.net/tabbycat/localized.svg)](https://crowdin.com/project/tabbycat)
 [![Docs](https://readthedocs.org/projects/tabbycat/badge/)](http://tabbycat.readthedocs.io/en/stable/)
-[![Build Status](https://travis-ci.org/TabbycatDebate/tabbycat.svg?branch=develop)](https://travis-ci.org/TabbycatDebate/tabbycat)
+![example workflow name](https://github.com/TabbycatDebate/tabbycat/workflows/django/badge.svg)
```

Comment:
Could have kept the "Build Status" alt-text.

Reply:
Yup just fixing that now - the badge itself is also not working